docs/gallery/howto/autogen/async.py

```diff
@@ -0,0 +1,68 @@
+"""
+Run tasks asynchronously
```

Comment:
All tasks in WorkGraph are run asynchronously, so maybe change to another title, e.g., `Run async functions as Tasks`